GI05MOAS-PG515/GI05MOAS-PG515_D00003_ingest.csv

```diff
@@ -3,6 +3,6 @@ mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_
 mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-PG515/D00003/merged-from-glider/gi_*.mrg,GI05MOAS-PG515-01-CTDGVM000,telemetered,Available,
 mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-PG515/D00003/merged-from-glider/gi_*.mrg,GI05MOAS-PG515-02-DOSTAM000,telemetered,Available,
 mi.dataset.driver.moas.gl.flort_m.flort_m_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-PG515/D00003/merged-from-glider/gi_*.mrg,GI05MOAS-PG515-03-FLORTM000,telemetered,Available,
-mi.dataset.driver.moas.gl.flort_o.flort_o_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-PG515/D00003/merged-from-glider/gi_*.mrg,GI05MOAS-PG515-04-FLORTO000,telemetered,Available,
+#mi.dataset.driver.moas.gl.flort_o.flort_o_glider_telemetered_driver,/omc_data/whoi/OMC/GI05MOAS-PG515/D00003/merged-from-glider/gi_*.mrg,GI05MOAS-PG515-04-FLORTO000,telemetered,Not Expected,Telem files do not contain data

```

Comment:
Looks good.  Per annotation 3347 the FLORT-O failed.

Reply:
looks good